### DISCUSSION TITLE
How to remove "Private information" fastest

### DISCUSSION THREAD

**Discussion by mbvb1223** [community/community#40253] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi all, 

I made a mistake by publishing a repo with some private information -> I have deleted my repo already.
But I found 2 forked repo from my repo -> these repos are still public.

I created a request to Github and get a response at `Nov 22, 2022, 11:17 PM (6 days ago) GT+7`
```
We’ve asked the repository owners to remove the private information you reported and have given them three business days to make changes or respond to the claim. We’ll let you know at that point if the changes have been made or what action we’ll be taking. 

You can add a comment by replying to this email.
```

But until now, it's been 6 days (more than 3 three business days) I have not gotten any news from Github.

I have replied the Github's email some times but still no signal from Git.

It is very **urgent** and **important** on my side.

Do you know any way to speed up it? Please advise!

Thanks

**Comment by airtower-luna** [comment, marked as answer]
> But until now, it's been 6 days (more than 3 three business days) I have not gotten any news from Github.

It's not more than three business days: Thursday was holiday in the US (Thanksgiving), and then there's the weekend. So business days after the 22nd are 23rd, 25th (Friday), and 28th (today).

**Comment by queenofcorgis** [comment]
Hi there - thank you for your patience, the Community is unable to escalate or interfere with tickets, but I checked on yours and it appears to be solved - is that so?
